prime-angular-frontend/src/app/modules/enrolment/pages/regulatory/regulatory-form-state.ts

```diff
@@ -1,18 +1,14 @@
-import { FormArray, FormBuilder, FormGroup } from '@angular/forms';
+import { FormArray, FormBuilder, FormGroup, FormControl } from '@angular/forms';
 import { CollegeCertification } from '@enrolment/shared/models/college-certification.model';
 
 import { AbstractFormState } from '@lib/classes/abstract-form-state.class';
 import { ConfigService } from '@config/config.service';
 import { CollegeLicenceClassEnum } from '@shared/enums/college-licence-class.enum';
+import { Enrolment } from '@shared/models/enrolment.model';
 
-export interface RegulatoryFormModel {
-  certifications: CollegeCertification[];
-}
+export interface EnrolmentRegulatoryForm extends Pick<Enrolment, 'certifications' | 'deviceProviderIdentifier'> { }
```

Comment:
Should be in it's own file `enrolment-regulatory-form.model.ts`

Reply:
fixed

Reply:
Should that file be in the same directory?

Reply:
Yes, same directory that the form-state is contained within